### PR TITLE
Create adewale2018.md

### PR DESCRIPTION
**Pull request by adewale2018**
Fixes #
  -  Create adewale2018.md

### RawGit preview link
 - [RawGit link]( https://raw.githack.com/adewale2018/adewale2018.github.io/add-my-MD-wikipage/#!pages/vi/profiles/adewale2018.md)

### CODE REVIEW

**Comment by singharpita** [comment]
@adewale2018 Link for your PR will be https://raw.githack.com/adewale2018/adewale2018.github.io/add-my-MD-wikipage/#!pages/vi/profiles/adewale2018.md

**Comment by adewale2018** [comment]
@singharpita, I could see you have approved the PR, what is the next step?
Thanks.

**Comment by singharpita** [comment]
@adewale2018 continue with first steps. Link is added there so that in future pull request you will write correct url. **rawgit** is no longer valid so we use **raw.githack**

**Comment by adewale2018** [comment]
OK, thanks

File: pages/vi/profiles/adewale2018.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+    I am from Nigeria. I studied mechanical engineering and was working in a flour mill company before I switched
+    to learning software development. I always look for opportunity to acquire the technical and the soft skills 
+    needed to become a world-class software engineer. It gladdens my heart to be on the process of becoming a 
+    member of this great community. Nice to meet you all!.
+```
+___
+##### `JOURNEY OF ALMOST A YEAR:`
+ - HTML5
+  - CSS3
```
Comment (svlesiv) [inline pages/vi/profiles/adewale2018.md]: I just noticed that indentation for the first element in the list is not the same as for other elements, and it creates a nested list. Could you please fix that.
Screen Shot 2019-04-09 at 11 19 35 AM

**Comment by adewale2018** [comment]
*@svlesiv, *now properly indented. Thanks

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request merged**